examples/duoswap/index.rsh

```diff
@@ -112,6 +123,8 @@ export const main = Reach.App(() => {
     parallelReduce([ true, initialMarket, 0 ])
       .define(() => {
         const st = [ alive, market ];
+        // Tokens.aBal.set(balance(tokA));
```

Comment:
The frontend isn't using this info right now and commenting out these views shave 2 minutes off of compilation time